dask/tests/test_distributed.py

```diff
@@ -388,3 +388,16 @@ async def test_annotation_pack_unpack(c, s, a, b):
     unpacked_hlg = HighLevelGraph.__dask_distributed_unpack__(packed_hlg, annotations)
     annotations = unpacked_hlg["annotations"]
     assert annotations == {"workers": {"n": ("alice",)}}
+
+
+@gen_cluster(client=True)
+async def test_map_partitions_partition_info(c, s, a, b):
+    dd = pytest.importorskip("dask.dataframe")
+    pd = pytest.importorskip("pandas")
```

Comment:
Unrelated to this PR, but maybe causing the failures in CI @jrbourbeau, I think there is something kind of wrong with using `importorskip` with `@gen_cluster(client=True)`. It might be cleaner to have a separate test files for dataframe/tests/test_distributed.py` and `array/tests/test_distributed.py` where the importorskip can just happen at the top of the file.